tests/Unit/Visualization/Python/Test_PlotDatFile.py

```diff
@@ -27,7 +27,7 @@ def setUp(self):
     def test_available_subfiles(self):
         with h5py.File(self.filename, 'r') as open_file:
             self.assertEqual(available_subfiles(open_file),
-                             ['/Group0/MemoryData.dat', '/TimeSteps2.dat'])
+                             ['Group0/MemoryData.dat', 'TimeSteps2.dat'])
```

Comment:
can we keep the `/` at the start?

Reply:
I could add it. It's not needed when passing the path to `--subfile-name` so I didn't bother 🤷‍♂️

Reply:
fair enough :)